### PR TITLE
[DOPS-1658]: Sora Subquery dev project. Set up autodeployment workflow

### PR DESCRIPTION
**Pull request by f33r0**
# Task

[DOPS-1658]: Sora Subquery dev project. Set up autodeployment workflow

## Changes

1. Change pipeline for redeploy dev environment on subquery

## Author

Signed-off-by: name <zaporozhets@soramitsu.co.jp>

[DOPS-1658]: https://soramitsu.atlassian.net/browse/DOPS-1658

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,11 +1,56 @@
...
+                       set -xe
+                       apt-get update && apt-get install --no-install-recommends unzip -y && apt-get install --no-install-recommends jq curl ca-certificates -y
+                       mkdir -p $HOME/.local/bin
+                       curl -LO https://github.com/fewensa/subquery-cli/releases/download/v${SUBQUERY_CLI_VERSION}/subquery-linux-x86_64.zip
+                       unzip subquery-linux-x86_64.zip -d $HOME/.local/bin/
+                       rm -rf subquery-linux-x86_64.zip
+                       /home/ubuntu/.local/bin/subquery --token ${SUBQUERY_TOKEN} deployment deploy \
+                           --org ${SUBQUERY_ORG} \
+                           --key ${matrix_chain} \
+                           --branch develop \
```
Comment (Cre-eD) [inline Jenkinsfile]: it will redeploy dev env, how will you redeploy test env?

File: Jenkinsfile [hunk 2]
```diff
@@ -1,11 +1,56 @@
 @Library('jenkins-library')
 
-def pipeline = new org.js.AppPipeline(
-    steps:              this,
-    test:               false,
-    dockerRegistryCred: 'bot-sora2-rw',
-    dockerImageName:    'sora2/subquery',
-    buildDockerImage:   'docker.soramitsu.co.jp/build-tools/node:14-ubuntu',
-    preBuildCmds:       ['yarn install', 'yarn codegen']
-)
```
Comment (Cre-eD) [inline Jenkinsfile]: You broke the CI for the app, but you should have created a new Jenkisfile-onfinality file

File: Jenkinsfile-onfinality [hunk 3]
```diff
@@ -0,0 +1,65 @@
+@Library('jenkins-library')
+
+String agentLabel        = 'docker-build-agent'
+String envImageName      = 'ubuntu:latest'
+String subqueryToken     = 'subquery-token'
+String targetEnvironment =  getEnvFromJobName()
+String jobList           = '../deploy/sora2-' + targetEnvironment
```
Comment (Cre-eD) [inline Jenkinsfile-onfinality]: image

Comment (Cre-eD) [inline Jenkinsfile-onfinality]:
```suggestion
String jobList           = '../sora2-' + targetEnvironment
```

**Review by Cre-eD — CHANGES_REQUESTED** [review]
fix comments

File: Jenkinsfile-onfinality (hunk 3 above)
Comment (f33r0) [reply Jenkinsfile-onfinality]: fix into jenkins

**Review by Cre-eD — APPROVED** [review]

**Pull request merged**